recipes/cocotb/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install -U --no-deps . -vvv --no-build-isolation
```

Comment:
```suggestion
$PYTHON -m pip install -U --no-deps . -vvv --no-build-isolation
```

Reply:
I can try but it didn't do anything before